sdk/lib/cicero/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
+list(APPEND SOURCE
+    cicarray.cpp
+    cicbase.cpp
+    ciccaret.cpp
+    cicfmap.cpp
+    cicimc.cpp
+    cicreg.cpp
+    cicuif.cpp)
+
+add_library(cicero_a STATIC ${SOURCE})
+add_library(cicero_w STATIC ${SOURCE})
+
+target_include_directories(cicero_a PRIVATE
+    ${REACTOS_SOURCE_DIR}/sdk/include
+    ${REACTOS_SOURCE_DIR}/sdk/include/ddk
+    ${REACTOS_SOURCE_DIR}/sdk/include/reactos
+    ${REACTOS_BINARY_DIR}/sdk/include/psdk)
+target_include_directories(cicero_w PRIVATE
```

Comment:
These include lists look a bit overcomplicated? :thinking: Our other static libraries (including win32 ones) don't do that, and we know they compile OK.

Reply:
Deleted.